extensions/amp-audio/amp-audio.md

```diff
@@ -67,9 +67,8 @@ For example:
 
 **autoplay**
 
-The `autoplay` attribute allows the author to specify when - if ever - the animated image will autoplay.
-
-The presence of the attribute alone implies that the animated image will always autoplay. The author may specify values to limit when the animations will autoplay. Allowable values are `desktop`, `tablet`, or `mobile`, with multiple values separated by a space. The runtime makes a best-guess approximation to the device type to apply this value.
+If present, the attribute implies that the audio start playing as soon as it is
```

Comment:
This is already merged, so I created https://github.com/ampproject/amphtml/pull/7583